pkg/apis/nmstate/v1alpha1/condition_types_test.go

```diff
@@ -103,6 +103,28 @@ var _ = Describe("Conditions list", func() {
 				Expect(newConditions).To(HaveLen(len(originalConditions)))
 			})
 
+			It("should change values and update LastTransitionTime and LastHearbeatTime", func() {
+				updatedCondition := newConditions.Find(originalConditions[0].Type)
+				Expect(updatedCondition.Status).To(Equal(updatedConditionStatus))
+				Expect(updatedCondition.Reason).To(Equal(updatedConditionReason))
+				Expect(updatedCondition.Message).To(Equal(updatedConditionMessage))
+				Expect(updatedCondition.LastTransitionTime.Time).To(BeTemporally(">", originalConditions[0].LastTransitionTime.Time))
+				Expect(updatedCondition.LastHeartbeatTime.Time).To(BeTemporally(">", originalConditions[0].LastHeartbeatTime.Time))
+			})
+		})
+		Context("and we reset it with unknown status and empty reason and message", func() {
```

Comment:
I don't see how is this Context needed. The `Set` does not care what you set, so the values in the previous context are as good as those from this Context. Am I wrong?

Reply:
Removing it, was introduced to check that it was working fine, but is not needed.